RevenueCat/Scripts/Purchases.cs

```diff
@@ -164,6 +164,28 @@ public void Setup(PurchasesConfiguration purchasesConfiguration)
     }
     
     // ReSharper disable once MemberCanBePrivate.Global
+    /// <summary>
+    /// Use this method to configure the SDK programmatically.
```

Comment:
Maybe mention that this should be called before any other SDK methods to work properly?

Reply:
Great call! I'll do that in a separate PR to avoid having to rebase all of these